### ISSUE TITLE
Fresh git annex breaks something in export archives test

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad-extensions#22]
Annex now builds and self tests but fails datalad in

```

======================================================================
ERROR: datalad.plugin.tests.test_export_archive.test_archive
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/datalad/tests/utils.py", line 548, in newfunc
    return t(*(arg + (d,)), **kw)
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/datalad/plugin/tests/test_export_archive.py", line 99, in test_archive
...
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/datalad/interface/utils.py", line 482, in return_func
testing 0
    results = list(results)
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/datalad/interface/utils.py", line 413, in generator_func
    allkwargs):
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/datalad/interface/utils.py", line 552, in _process_results
    for res in results:
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/datalad/plugin/export_archive.py", line 168, in __call__
    **(tar_args if archivetype == 'tar' else {}))
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/tarfile.py", line 1926, in add
    tarinfo = self.gettarinfo(name, arcname)
  File "/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/tarfile.py", line 1805, in gettarinfo
    statres = os.lstat(name)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/datalad_temp_tree_test_archiveunoamzm6/ds/.git/annex/objects/VW/Qq/MD5E-s12--baab6c16d9143523b7865d46896e4596/MD5E-s12--baab6c16d9143523b7865d46896e4596
```

**Comment by yarikoptic** [comment]
@kyleam could you look into it while it is "hot" or should I ?

**Comment by kyleam** [comment]
For posterity, could you provide links to the actual build?

I'll look into it at some point this week.

**Comment by yarikoptic** [comment]
Just a note that `tools/ci/install-annex.sh datalad-extensions-build` will not fetch the artifact of the overall failed build... Filed https://github.com/datalad/datalad/issues/4740

**Comment by kyleam** [comment]
I can trigger it locally and have bisected it.   I look into it more a bit later.

**Comment by kyleam** [comment]
[Filed upstream](https://git-annex.branchable.com/bugs/find_--batch_reports_files_that_have_no_content/)

**Comment by kyleam** [comment]
Fixed by Joey with abd56fb01 (Fix a bug in find --batch in the previous version, 2020-07-20) and released as 8.20200720.1.

The latest build is green: https://github.com/datalad/datalad-extensions/actions/runs/176470285
